en/translator-guidelines.md

```diff
@@ -52,12 +52,12 @@ Our preferred way for translators to submit a lesson is to add them directly to
 ### Translation Submitted! Now What?
 To see what happens after you submit a translation, feel free to browse our [editor guidelines](/editor-guidelines), which detail our editorial process. Highlights are below:
 
-The most immediately important step is that your editor will create an [issue](https://github.com/programminghistorian/ph-submissions/issues) for the new translation on the [ph-submissions] repository, with a link to your lesson (that you previewed in step 5). The editor and at least two reviewers invited by the editor will post their comments to this issue.
+The most immediately important step is that your editor will create an [issue](https://github.com/programminghistorian/ph-submissions/issues) for the new translation on the [ph-submissions](https://github.com/programminghistorian/ph-submissions) repository, with a link to your lesson (that you previewed in step 5). The editor and at least two reviewers invited by the editor will post their comments to this issue.
 
 ### Wait for Reviewer Feedback
 We aim to complete the review process within four weeks, but sometimes delays occur or people get busy and the process can take longer than we hoped.
 
-In keeping with the ideas of public scholarship and open peer review, we encourage discussions to stay on GitHub. However, we also want everyone to feel comfortable with the process. If you need to discuss something privately, please feel free to [email your editor directly](/project-team), or to contact one of our dedicated ombudsperson (Dr Ian Milligan - i2milligan@uwaterloo.ca).
```

Comment:
In case you do not want the email address of the ombuds to be visible on the page, you may enter a link like: [Dr Ian Milligan](i2milligan@uwaterloo.ca)